code/headers/temperature_frame_action.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <frame_action.hpp>
+#include <hwlib.hpp>
+
+namespace r2d2::robos {
+    class temperature_frame_action_c
+        : public frame_action_c<frame_type::TEMPERATURE> {
+    private:
+        // Display frame types
+        frame_cursor_color_s cursor_color;
```

Comment:
Check cpp comments

Reply:
fixed in https://github.com/R2D2-2019/RobOS/pull/2/commits/87c92d1dc8e85dba1e0b57089281459eaa30efef